test/cmd/events.sh

```diff
@@ -61,6 +117,10 @@ run_kubectl_events_tests() {
     output_message=$(kubectl events -n test-events --for Cronjob.v1.batch/pi "${kube_flags[@]:?}" 2>&1)
     kube::test::if_has_string "${output_message}" "Warning" "InvalidSchedule" "Cronjob/pi"
 
+    # Post-Condition: events not returns event for fully qualified Cronjob.v1.example.com/pi when --for flag is used
+    output_message=$(kubectl events -n test-events --for Cronjob.v1.example.com/pi "${kube_flags[@]:?}" 2>&1)
+    kube::test::if_has_not_string "${output_message}" "Warning" "InvalidSchedule" "Cronjob/pi"
```

Comment:
Did you verify that this test is failing on master(without the patch)?

Reply:
I did some testing locally. Before I made the changes, running 'kubectl events -n test-events --for Cronjob.v1.example.com/pi` returned the events for Cronjob.v1.batch. After I made my changes, running `kubectl events -n test-events --for Cronjob.v1.example.com/pi` returned no events, which is the expected result.